### ISSUE TITLE
Graph2d: Unable to flip Y-Axis

### ISSUE THREAD

**Issue by farhanansari** [almende/vis#2131]
Went though the documentation here: http://visjs.org/docs/graph2d/#graph2dOptions but unable to find a way to flip Y-Axis.  I understand that it will also flip X-Axis which is fine.  Following is a very simple code Im using:

`                    var dataset = new vis.DataSet(json.result);
                    var options = {
                        start: moment().subtract(1, 'month').format('YYYY-MM-DD'),
                        end: moment().format('YYYY-MM-DD'),
                    };

                    var graph2d = new vis.Graph2d(container, dataset, options);`

I tried playing with 'sort' option but that didn't help either.  Do we have any way to do this? 

<img width="1108" alt="screen shot 2016-10-03 at 7 46 54 pm" src="https://cloud.githubusercontent.com/assets/11396965/19061099/a6f49ed4-89a2-11e6-841e-2d50a4f2a222.png">

Basically I am trying to get my Y-Axis to start from 10 on top and go all the way down to 26 at the bottom.

Appreciate your help on this.
